tests/cases/dom/manipulate/empty.js

```diff
@@ -0,0 +1,65 @@
+import QUnit from "qunitjs";
+import {empty} from "../../../../dom/manipulate";
+
+QUnit.module("dom/manipulate/empty()",
+    {
+        beforeEach: () =>
+        {
+            document.getElementById("qunit-fixture").innerHTML = `
+                <p id="test-element" class="test">content</p>
+                <p id="test-element-two" class="test">content</p>
+            `;
+        },
+    }
+);
+
+
+QUnit.test(
+    "with valid node element",
+    (assert) =>
+    {
+        const element = document.getElementById("test-element");
+        empty(element);
+
+        const result = document.getElementById("test-element");
+        assert.equal(result.innerHTML, "", "element is empty");
```

Comment:
The first assertion should be that the elements are not empty before `empty()` was called.